lxc/image.go

```diff
@@ -389,41 +388,22 @@ func (c *imageCmd) run(conf *config.Config, args []string) error {
 			return err
 		}
 
-		// Optimisation for simplestreams
-		var imgInfo *api.Image
+		// Attempt to resolve an image alias
 		image := inName
-		if conf.Remotes[remote].Protocol == "simplestreams" {
-			imgInfo = &api.Image{}
-			imgInfo.Fingerprint = image
-			imgInfo.Public = true
-		} else {
-
-			// Attempt to resolve an image alias
-			alias, _, err := d.GetImageAlias(image)
-			if err == nil {
-				image = alias.Target
-			}
-
-			// Get the image info
-			imgInfo, _, err = d.GetImage(image)
-			if err != nil {
-				return err
-			}
```

Comment:
What bothers me here is that you're now adding a 10s lag for any case where we're using cloud images even if copy-aliases isn't specified.

GetImage is very expensive on simplestream remotes which is why we had the optimization that you have now removed.